src/react-decathlon/src/App.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react'
+import { MeiliSearch } from 'meilisearch'
+import Item from './components/Item'
+
+const client = new MeiliSearch({
+  host: 'http://127.0.0.1:7700/'
+})
+
+const index = client.index('decathlon')
+
+function App () {
+  const [searchedWord, setSearch] = useState('')
+  const [resultSearch, setResults] = useState([])
+
+  useEffect(() => {
+    // Create an scoped async function in the hook
+    async function searchWithMeili () {
+      const search = await index.search(searchedWord, {
+        limit: 24,
+        attributesToHighlight: ['name']
+      })
+      setResults(search.hits)
+    }
+    // Execute the created function directly
+    searchWithMeili()
+  }, [searchedWord])
+
+  return (
+    <div className='mx-auto'>
+      <div className='header font-sans text-white items-center justify-center'>
+        <header className='py-12'>
+          <img
+            className='h-20 w-auto items-center justify-center p-2 mx-auto'
+            src='/wide_logo.png'
+            style={{ filter: 'invert(0%)' }}
+            alt=''
```

Comment:
Can we add an `alt` description here? If not, it's better to remove it :)
Or maybe you want to keep it that way so that people can `diff` the file and see only the changes that are relevant? 🤔